pkg/api/validation/validation.go

```diff
@@ -671,6 +679,25 @@ func validateNFSVolumeSource(nfs *api.NFSVolumeSource, fldPath *field.Path) fiel
 	return allErrs
 }
 
+func validateQuobyteVolumeSource(quobyte *api.QuobyteVolumeSource, fldPath *field.Path) field.ErrorList {
+	glog.V(4).Infof("quobyte vol %#v registry %#v", quobyte.Volume, quobyte.Registry)
+	allErrs := field.ErrorList{}
+	if len(quobyte.Registry) == 0 {
+		allErrs = append(allErrs, field.Required(fldPath.Child("registry"), "Quobyte registry can't be nil"))
+	} else {
+		for _, hostPortPair := range strings.Split(quobyte.Registry, ",") {
+			if _, _, err := net.SplitHostPort(hostPortPair); err != nil {
+				allErrs = append(allErrs, field.Required(fldPath.Child("registry"), fmt.Sprintf("Quobyte registry %v", err)))
+			}
+		}
+	}
+
+	if len(quobyte.Volume) == 0 {
+		allErrs = append(allErrs, field.Required(fldPath.Child("volume"), "Quobyte volume can't be nil"))
```

Comment:
Again, "Quobyte volume" is redundant here.  "nil" is not a concept in the JSON API, please don't say nil.  The fact that this is a field.Required() error will already indicate that the field is required but not provided, so just leave the details field blank, unless you have more information to add (whcih you don't)